src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -49,7 +49,6 @@ module ocn_gm
    logical :: local_config_GM_compute_visbeck
    logical :: local_config_GM_lat_variable_c2
```

Comment:
These local config variables don't have any explanation of what they do. Most are clear, but would be good to have a sentence on this one. Does it mean that c2 varies with latitude? You could also put the explanation just before this line:
```
500 if (local_config_GM_lat_variable_c2) then
```

Reply:
My recollection is you and I decided to put them in so we could just have GM configurations in the namelist and not lots of options.  Although that only explains the second, not sure why I put a local_config_GM-compute_visbeck in there though.  Let me clean this up

Reply:
I think it's fine to have `local_config` variables if they are not meant to be changed by the user. I was saying that there should be a comment explaining what it is, either here, or where it is used in the code.

Reply:
@mark-petersen a question for you.  Does it matter performance wise if we do a logical comparison for if statements or a string compare?  We could remove these local_configs and just use things like
```
if (config_gm_closure == 'visbeck')
```
but not sure which is better practice

Reply:
makes sense, I agree on that.  Let me know what you think about my question just above and I'll add comments and make the other fixes.

Reply:
Logical comparisons always better. So either logical flags or if there are multiple choices, setting up enum-like integer parameters as choices are much preferred. Strings must compare every letter and deal with a variety of conditions, terminations, etc.

Reply:
@philipwjones is a string compare actually slow enough to matter if it is outside of a cell or edge loop? If so, we can do the string compare on init and replace it with a local logical or index. Doug used to do that, but it leads to confusing code because you need to always trace through extra variables.

Reply:
@mark-petersen I think the code right now does the string compare on init and then sets the logicals right?